apps/passport-server/src/services/userService.ts

```diff
@@ -352,9 +348,6 @@ export class UserService {
       extra_issuance: false
     });
 
-    // Reload Merkle trees
-    this.semaphoreService.scheduleReload();
```

Comment:
why are we not reloading anymore?

Reply:
semaphore service is one of those services that continues to be run on the main process, whereas user service is split up amongst the cluster processes. thus they can't directly communicate like they were before here. 

the reason this change is ok anyways is that previously (during zuzalu) it was super important that the merkle tree updates super quickly after an account is created, because zupoll was basically the only app. nowadays this is less important, and the semaphore groups are updated eventually anyways independently of them being triggered to update by account creation.